docs/project/list-of-diagnostics.md

```diff
@@ -78,6 +78,7 @@ The PR that reveals the implementation of the `<IncludeInternalObsoleteAttribute
 |  __`SYSLIB0021`__ | Derived cryptographic types are obsolete. Use the Create method on the base type instead. |
 |  __`SYSLIB0022`__ | The Rijndael and RijndaelManaged types are obsolete. Use Aes instead. |
 |  __`SYSLIB0023`__ | RNGCryptoServiceProvider is obsolete. To generate a random number, use one of the RandomNumberGenerator static methods instead. |
+|  __`SYSLIB0024`__ | Creating and unloading AppDomains is not supported and will throw PlatformNotSupportedException. |
```

Comment:
I augmented the message to indicate that PNSE will be thrown.

Reply:
I tried to not to say that because `AppDomain.Unload` will always throw `CannotUnloadAppDomainException`, which doesn't inherit from PNSE.

Reply:
Ah; good point. How about "...and will throw exceptions" instead?

Reply:
```suggestion
|  __`SYSLIB0024`__ | Creating and unloading AppDomains is not supported and throws exceptions. |
```

Reply:
```suggestion
|  __`SYSLIB0024`__ | Creating and unloading AppDomains is not supported and throws an exception. |
```

"Throws an exception" sounds better to me.